src/Targets/Microsoft.Managed.DesignTime.targets

```diff
@@ -32,7 +32,7 @@
     <ProjectCapability Include="UseFileGlobs"/>
     <ProjectCapability Include="EditAndContinue"/>
     <ProjectCapability Include="OpenProjectFile"/>
-    <ProjectCapability Include="GenerateNuGetPackage"/>
+    <ProjectCapability Include="Pack"/>
```

Comment:
Let's only merge this change in when the NuGet issue is fixed. Otherwise the pack package will disappear.